OpenRA.Mods.Common/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -12,13 +12,16 @@
 using System;
 using System.Linq;
 using OpenRA.Activities;
-using OpenRA.Mods.Common;
 using OpenRA.Mods.Common.Activities;
 using OpenRA.Primitives;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum EntryType { Fixed, PlayerSpawnClosestEdge, DropSiteClosestEdge }
```

Comment:
As clarified in commit 75a0cb44222d7be5e610bebd62cf99cdd3574cb0 `PlayerSpawnClosestEdge` isn't really spawning at the closest edge per se, neither was that the original intent. This name should probably be fixed. It would also be good if there was a proper description for all the different options here and for `ExitType`.

Reply:
Ah that makes sense. Will change.

Reply:
Scratch that. I don't understand what it's doing. Can you give me a name and a description for it and I'll add them along with descriptions for the rest?

Reply:
Alright, how about: `PlayerBaseline` "Spawns at the map edge directly behind the player spawn point (from the perspective of the map center)"

A bit of context: The idea here was inspired by some tabletop games where special significance is given to the edge of the board that is physically closest to the player. Sometimes called a baseline or home-row  or something like that. In those games reinforcements can only be placed at your map edge (or sometimes units must exit through the _opponent's_ baseline to gain an advantage, like checkers).

In order to make the concept applicable to the wide variety of possible spawn point configurations (including FFAs) the baselines are defined radially, like pizza slices. This means that if opponents spawn in opposing corners, the reinforcements will arrive along the diagonal of the map and in Tiberium Garden type maps, the reinforcement locations are appropriately separated.

Reply:
I think I get the idea, but testing on the map Arcade threw me off. Thanks. I'll try to update along with everything else shortly.